sentry_streams/sentry_streams/adapters/arroyo/steps.py

```diff
@@ -151,3 +152,21 @@ def extract_value(message: Message[Union[FilteredPayload, RoutedValue]]) -> Any:
             extract_value,
             Produce(self.producer, Topic(self.topic_name), next),
         )
+
+
+@dataclass
+class ReduceStep(ArroyoStep):
+
+    pipeline_step: Reduce[Any, Any, Any]
+
+    def build(
+        self, next: ProcessingStrategy[Union[FilteredPayload, RoutedValue]]
+    ) -> ProcessingStrategy[Union[FilteredPayload, RoutedValue]]:
+
+        windowed_reduce: ProcessingStrategy[Union[FilteredPayload, RoutedValue]] = (
+            build_arroyo_windowed_reduce(
+                self.pipeline_step.windowing, self.pipeline_step.aggregate_fn, next, self.route
```

Comment:
Are you planning to support group by keys in the future ?

Reply:
yes